packages/controller/src/controller.ts

```diff
@@ -112,7 +111,7 @@ export default class ControllerProvider extends BaseProvider {
 
     try {
       let response = await this.keychain.connect(
-        this.options.policies || [],
+        this.options.policies || {},
```

Comment:
Can we avoid making any backwards incompatible changes to `packages/controller`?